ably/proto/message_test.go

```diff
@@ -172,4 +230,40 @@ var _ = Describe("Message", func() {
 			})
 		})
 	})
+
+	acceptance := func(fixture string, t GinkgoTInterface) {
+		test, cfg := load(fixture, t)
+
+		It("fixture decode", func() {
+			for _, item := range test.Items {
+				err := item.Encoded.DecodeData(nil)
+				Expect(err).NotTo(HaveOccurred())
+				err = item.Encrypted.DecodeData(cfg)
+				Expect(err).NotTo(HaveOccurred())
+				Expect(item.Encrypted.Name).To(Equal(item.Encoded.Name))
+				Expect(item.Encrypted.Data).To(Equal(item.Encoded.Data))
+			}
+		})
+
+		return
```

Comment:
Oh now I remember why I even need this - the fixture had a bug. I've sent https://github.com/ably/ably-common/pull/3. After it gets merge I'll update submodule in ably-go. In the meantime the tests might fail, an honest disclaimer ;)